pkg/idhubmain/commitroot.go

```diff
@@ -22,14 +20,11 @@ func RunCommitRoot() error {
 		log.Fatalf("error initializing eth helper: %v", err)
 	}
 
-	rootCommitter, err := claims.NewRootCommitter(ethHelper, ethHelper.Blockchain.(ethereum.TransactionReader), config.RootCommitsAddress)
-	if err != nil {
-		log.Fatalf("error initializing root committer: %v", err)
-	}
+	log.Fatalf("error initializing root committer: %v", err)
```

Comment:
If you call `log.Fatalf`, the process will exit, is this intended here?  Seems that this error log shouldn't be here?

Reply:
yeah, i messed up while refactoring and this slipped through